client/app/templates/components/game-client.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-12 console">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span class="glyphicon glyphicon-comment"></span> Client
+        </div>
+        <div class="panel-body">
+          <ul class="chat">
+            {{#each messages as |msg|}}
+              <li class="clearfix">
+                <div class="chat-body clearfix">
+                  {{#if msg.server}}
```

Comment:
Ah, good point could do that... cool yeah I'm good with that. Because the rest is just css so cool. Sorry miss understood. I will try to do that tonight then if I get a chance I think it's worth doing and quick.

Other than that are you ok with the look of it for now? We can style as we go along of course but seems to be somewhat in the right vein.